app/uk/gov/hmrc/nisp/controllers/AccountController.scala

```diff
@@ -91,8 +91,15 @@ trait AccountController extends NispFrontendController with AuthorisedForNisp wi
                 spSummary.forecast.forecastAmount,
                 spSummary.forecast.personalMaximum
               )
-            Ok(account(nino, spSummary, currentChart, forecastChart, personalMaximumChart, isPertax))
-               .withSession(storeUserInfoInSession(user, spSummary.contractedOutFlag))
+            Ok(account(
+              nino,
+              spSummary,
+              currentChart,
+              forecastChart,
+              personalMaximumChart,
+              isPertax,
+              futureProofPersonalMax = applicationConfig.futureProofPersonalMax && spSummary.numberOfGaps > 1
```

Comment:
It seems strange that you're combining the config property and number of gaps rule into something named the same as the config property?